tests/components/side-panel/side-panel-header.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import { render, screen } from "@testing-library/react";
+import "jest-canvas-mock";
+import { SidePanelHeader } from "src/components/side-panel/side-panel-header";
+import { EBuilderMode, EElementType } from "src/context-providers";
+import { ELEMENT_BUTTON_LABELS } from "src/data/elements-data";
+import { TestHelper } from "src/util/test-helper";
+
+describe("side-panel-header", () => {
```

Comment:
apologies, I just notice the incorrect naming, may I request to use the component name as the description of the main test suite, so this should be `SidePanelHeader`

do apply this to the rest of the test suites including the recently approved ones in `element-card`